dbms/src/IO/Compression/CompressionCodecMultiple.h

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <IO/Compression/ICompressionCodec.h>
+
+namespace DB
+{
+
+
+/** Compression codec for multiple codecs.
+  * It compresses data with multiple codecs and stores the list of codecs in the header.
+  * | <---------- Compression Header ---------> | <------------------------------------------- Additional Data -----------------------------------------------------------> | <- Compressed Data -> |
```

Comment:
Do we have a version label here?

Reply:
IMO, this format should never be changed, so it is better to keep it simple, we do not need any version label.

Even though for some special reasons, we need to change this, just add a new codec.